packages/jsii-dotnet-analyzers/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euo pipefail
+
+dotnet build --force -c Release ./src/Amazon.JSII.Analyzers
+
+cp -f ./bin/Release/NuGet/*.nupkg .
```

Comment:
Should this be copied to `dist` instead?

Reply:
I based this file on the jsii-dotnet-runtime's one. It does not have a dist directory?